frb/associate/dev/utils.py

```diff
@@ -0,0 +1,119 @@
+""" Codes realted to developing what has become PATH (aka astropath)"""
+
+import pdb; pdb.set_trace()  # This module is defunct
+
+def add_contam(ncontam, frb_coord, cand_gal, cand_coord, fov,
```

Comment:
Add comment specifying units for fov, ra, dec

Reply:
this is basically deprecated code